src/krati_Shukla/Exam1/SumOfNos.java

```diff
@@ -0,0 +1,36 @@
+package krati_Shukla.Exam1;
+
+public class SumOfNos {
+	
+	int[] TargetIndex(int[] input, int target) {
+		int[] output = new int[2];
+			
+		int length = input.length;
+		for(int index=0; index<length; index++) {
+			for(int newIndex=1; newIndex<length; newIndex++) {
```

Comment:
Inner for loop should from i+1, think on this, if you want to connect on Skype then feel free to message.

Take reference of this code
Kangan - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/987/files
Rupali - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/991/files